test/fixtures/documents.yml

```diff
@@ -8,8 +8,32 @@ background:
 
 cat_photo:
   user: keith
-  assistant: samantha
+  assistant:
```

Comment:
I am pretty sure that documents should be connected to the user -or- the assistant. I didn't do testing of the OpenAI API to be sure and the docs weren't totally clear so I just fixed it here in the fixtures. But if that is true then we should probably make this association polymoprhic.